.changeset/moody-kings-hope.md

```diff
@@ -0,0 +1,25 @@
+---
+'@primer/react-brand': minor
+---
+
+Added new Label component
+
+Use `Label` to indicate the status of the content or add metadata to the `Card` component.
+
+```jsx
+<Label color="green" size="large">
+  New feature
+</Label>
+```
+
+:link: [See the documentation for more details and usage examples.](https://primer.style/brand/components/Label)
+
+Update Card component to use Label component and its color variants
+
+```jsx
+<Card href="https://github.com">
+  <Card.Label color="green">New feature</Card.Label>
+  <Card.Heading>Collaboration is the key to DevOps success</Card.Heading>
+  <Card.Description>Everything you need to know about getting started with GitHub Actions.</Card.Description>
+</Card>
+```
```

Comment:
Could we also add a separate changeset for `@primer/brand-primitives` please? 

E.g of how it's previously been done:

https://github.com/primer/brand/blob/main/packages/design-tokens/CHANGELOG.md#0150

So should contain a list a tokens specific to the Label